app/ForkedStar.tsx

```diff
@@ -0,0 +1,73 @@
+// const { Octokit } = require("@octokit/action");
+const { Octokit } = require("octokit"); // Use this for local development
+import { env } from "@/env.mjs";
```

Comment:
Why? Slack EOD Reminder is not part of the NextJS app and is part of a GitHub Workflow. 

Use the pre-authenticated instance of octokit. See https://github.com/coronasafe/leaderboard/blob/main/lib/octokit.ts#L19 or other places where octokit is used in the repository (eg: https://github.com/coronasafe/leaderboard/blob/main/app/feed/page.tsx#L18)
```suggestion
```